analyze_text.py

```diff
@@ -1,4 +1,21 @@
-# Remember to reach out for help after your own due diligence
-
 def analyze_text(text):
-    # Your code here
+    return "The text contains " + str(letter_count(text)) + " alphabetic characters, of which " + str(e_count(text)) + " (" + str(percentifier(letter_count(text), e_count(text))) + "%) are 'e'."
+
+def letter_count(text):
+    letter_counter = 0
+    for char in text:
+        if char.isalpha():
+            letter_counter += 1
+    return letter_counter
+
+def e_count(text):
+    e_counter = 0
+    for char in text:
+        if char == "e" or char == "E":
+            e_counter += 1
+    return e_counter
+
+def percentifier(letter, e):
+    percent = (e / letter) *100
+    #return '{:.2%}'.format(percent)
```

Comment:
You had the right idea to format this to two places. Try uncommenting this return statement and comment the other one. I believe that is why your tests are failing.